summarization/summarizer_main.py

```diff
@@ -40,6 +41,19 @@ def __init__(
         # is about one broad topic or multiple small topics
         self.title_topic_is_plural = title_topic_is_plural
 
+        self._strip_title_and_make_stamp()
+
+    def _strip_title_and_make_stamp(self):
+        if self.CAPPING_METHOD == "interesting-sequence-picker":
+            # we only need the seed title if we use
+            # interesting sequence picker
+            # we put it as the first member of the list
+            self._assemble_and_add_stamp_pages_to_list(
+                [self.contents.title_text[0]]
+            )
+
+        self.contents.title_text.pop(0)
+
```

Comment:
Didn't understand this method completely.

Reply:
The first item in the title_text will be the webpage title - apologies for this maybe i should add a comment explaining the same!

Reply:
added comments explaining the same! :D